lib/ui.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+# module for user interface
+module UI
+  PLAYER1MSG = "Player1 won\nWould you like to play again? Type Y/N"
+  PLAYER2MSG = "Player2 won\nWould you like to play again? Type Y/N"
+  DRAWMSG = "It's a draw\nWould you like to play again? Type Y/N"
+  def instructions
+    puts 'Tictactoe game is 3X3 board game. You need to choose either X or O'
+    puts '******************************************************************'
+    puts '---Every turn the players puts the sign X or O on their turn if---'
+    puts '******************************************************************'
+    puts '----Any player gets 3 consequetive X or O then he or she wins-----'
+  end
+
+  def show_board(board)
```

Comment:
I'm sure you guys can find a way to test this method.

Hint: Try testing stdout expected

Reply:
The change for this method has been done